Sources/SPTDataLoader/SPTDataLoaderService.m

```diff
@@ -138,6 +139,7 @@ - (instancetype)initWithConfiguration:(NSURLSessionConfiguration *)configuration
 
         _fileManager = [NSFileManager defaultManager];
         _dataClass = [NSData class];
+        _sessionInvalidated = NO;
```

Comment:
nit: this is redundant as it's `NO` by default

Reply:
I like to be explicit, if that is not an issue to have it there